yapf/yapflib/style.py

```diff
@@ -423,5 +432,5 @@ def _CreateStyleFromConfigParser(config):
 # TODO(eliben): For now we're preserving the global presence of a style dict.
```

Comment:
It is not good solution to make non-constant variable global for whole project.
Hope my solution is appropriate in this situation.